src/Environment.swift

```diff
@@ -0,0 +1,150 @@
+import Foundation
+
+/// Access workflow environment variables
+///
+/// See the [Alfred Workflow Documentation](https://www.alfredapp.com/help/workflows/script-environment-variables/) for more info.
+public enum Env {
+    /// Location of the Alfred.alfredpreferences.
+    ///
+    /// Example output: "/Users/Crayons/Dropbox/Alfred/Alfred.alfredpreferences"
+    public static var preferences: String? {
+        return envvar("alfred_preferences")
+    }
+
+    /// Location of local (Mac-specific) preferences.
+    ///
+    /// Example output: "adbd4f66bc3ae8493832af61a41ee609b20d8705"
+    public static var localPreferences: String? {
+        guard
+            let preferences = Env.preferences,
+            let localHash = envvar("alfred_preferences_localhash")
+        else {
+            return nil
+        }
+
+        return "\(preferences)/preferences/local/\(localHash)"
+    }
+
+    /// Current Alfred theme.
+    ///
+    /// Example output: "alfred.theme.yosemite"
+    public static var theme: String? {
+        return envvar("alfred_theme")
+    }
+
+    /// Theme background color.
+    ///
+    /// Example output: "rgba(255,255,255,0.98)"
+    public static var themeBackground: String? {
+        // TODO: Is the string's format always the same? If so it would make sense to return a color value directly.
+        return envvar("alfred_theme_background")
+    }
+
+    /// Theme's selected item background color.
+    ///
+    /// Example output: "rgba(255,255,255,0.98)"
+    public static var themeSelectionBackground: String? {
```

Comment:
Would be cool to have a convenience getter for the UIColor value of this and `themeBackground`, wouldn't it?